src/Generator/FileGenerator.php

```diff
@@ -336,7 +336,7 @@ public function setClasses(array $classes)
 
     /**
      * @param  string $name
-     * @return ClassGenerator
+     * @return ClassGenerator|false
```

Comment:
false is not data type, use `bool` instead

Reply:
`false` is acceptable since PHP 8 and in phpstan/psalm context, but we should really add an assertion instead of widening the type (which is a BC break)

Reply:
Well, i changed it to bool now.
But feel free to set it to false if you prefer that, either way it fixes my phpstan issue :)